c7n/filters/backup.py

```diff
@@ -0,0 +1,72 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from .core import Filter
+from datetime import datetime, timedelta
+from c7n.utils import type_schema, local_session, chunks
+from c7n.query import RetryPageIterator
+
+
+class ConsecutiveAwsBackupsFilter(Filter):
+    """Returns resources where number of consective daily backups is
```

Comment:
actually one more bit @jasgrover could you clarify that this filter is for AWS backups as there's another filter (consecutive-backups) that uses the native backups?